pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -194,7 +212,10 @@ func (p *criStatsProvider) getFsInfo(storageID *runtimeapi.StorageIdentifier) *c
 	return &fsInfo
 }
 
-func (p *criStatsProvider) makePodStats(podSandbox *runtimeapi.PodSandbox) *statsapi.PodStats {
+func (p *criStatsProvider) makePodStats(
```

Comment:
I prefer keep the original function. Add another function to overwrite the original result.
In this way, it's easier to remove either part in the future.

Reply:
done